### DISCUSSION TITLE
Dark mode specific defaults

### DISCUSSION THREAD

**Discussion by kripod** [tailwindlabs/tailwindcss#2720] [Ideas]
While `borderColor` is set to `'colors.gray.200'` by default, that may be too light when using dark mode. There should be a way to override `DEFAULT` values for that case, e.g.:

```js
/* tailwind.config.js */

module.exports = {
  theme: {
    extends: {
      darkMode: {
        borderColor: (theme) => ({
          DEFAULT: theme('colors.gray.800', 'currentColor'),
        }),
      },
    },
  },
};
```

As an alternative, a `DEFAULT_DARK` key may be added and applied if available.

**Comment by simonswiss** [comment]
Hi!

That's a good idea, but it would be somewhat complicated to implement because we have to respect the `darkMode` strategy. I would say it's not worth adding the complexity to the config, and probably better to specify the border-colour every time you use a border utility.
